src/pages/iou/IOUDescriptionPage.js

```diff
@@ -0,0 +1,88 @@
+import React, {Component} from 'react';
+import {View} from 'react-native';
+import {withOnyx} from 'react-native-onyx';
+import PropTypes from 'prop-types';
+import TextInput from '../../components/TextInput';
+import withLocalize, {withLocalizePropTypes} from '../../components/withLocalize';
+import ScreenWrapper from '../../components/ScreenWrapper';
+import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
+import Form from '../../components/Form';
+import ONYXKEYS from '../../ONYXKEYS';
+import styles from '../../styles/styles';
+import Navigation from '../../libs/Navigation/Navigation';
+import compose from '../../libs/compose';
+import * as IOU from '../../libs/actions/IOU';
+
+const propTypes = {
+    ...withLocalizePropTypes,
+
+    /** Onyx Props */
+    /** Holds data related to IOU view state, rather than the underlying IOU data. */
+    iou: PropTypes.shape({
+        comment: PropTypes.string,
+    }),
+};
+
+const defaultProps = {
+    iou: {
+        comment: '',
+    },
+};
+
+class IOUDescriptionPage extends Component {
```

Comment:
NAB should we create this as a functional component and use hooks already?

Reply:
I'm not up to speed on hooks fully so I didn't go for it just for the sake of time. What do you think? Maybe a follow-up?

Reply:
Seems fine, but I'm still not sure why this is a class component since we don't have any state or lifecycle methods. Seems like we could refactor to a functional component without the need for hooks.